### PR TITLE
Update readme to reflect current progress

### PR DESCRIPTION
**Pull request by LoganParker**
- More so an update of the description of the project and our current progress - update regarding API is still required
- related to #244

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,11 +100,13 @@ charge of mapping spoken language from an audio file into a series of phonemes,
 distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
 receives the user text file and language selection.
 
-Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor
-maps phonemes from the phoneme file to mouth assets received from a directory. There will be 1:1 mapping between
-phonemes and mouth assets. These assets will be added onto the avatar and rendered into a video file. The video file
-will be sent back to the front end and will be available for the user to download onto their local machine.
+Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor maps phonemes from the phoneme file
```
Comment (vnajack) [inline README.md]:
```suggestion
Once the phoneme file is created, it is sent to the secondary process called the phoneme processor. This processor maps phonemes from the phoneme file 
```

File: README.md [hunk 2]
```diff
@@ -104,11 +100,13 @@ charge of mapping spoken language from an audio file into a series of phonemes,
 distinguish one word from another. In order to make this process as accurate as possible, the phoneme factory also
 receives the user text file and language selection.
 
-Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor
-maps phonemes from the phoneme file to mouth assets received from a directory. There will be 1:1 mapping between
-phonemes and mouth assets. These assets will be added onto the avatar and rendered into a video file. The video file
-will be sent back to the front end and will be available for the user to download onto their local machine.
+Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor maps phonemes from the phoneme file 
+to a specific timestamp, and associates that timestamp with an asset file containing the phoneme, an avatar pose, and state of blinking. There is a 1:1 
+mapping between phonemes and mouth assets. These assets are then compiled and rendered into a video file using ffmpeg. The video file is then returned to
```
Comment (vnajack) [inline README.md]:
```suggestion
mapping between phonemes and mouth assets. These assets are then compiled, realism is added for the rest of the body, and it is all rendered into a video file using ffmpeg. The video file is then returned to 
```

File: README.md [hunk 3]
```diff
@@ -104,11 +100,13 @@ charge of mapping spoken language from an audio file into a series of phonemes,
...
+Once the phoneme file is created, it is sent off to a secondary process called the phoneme processor. This processor maps phonemes from the phoneme file 
+to a specific timestamp, and associates that timestamp with an asset file containing the phoneme, an avatar pose, and state of blinking. There is a 1:1 
+mapping between phonemes and mouth assets. These assets are then compiled and rendered into a video file using ffmpeg. The video file is then returned to 
+the front end, and is available for the user to download onto their local machine.
 
 ## Limitations
 
-TBD
+- The phoneme generation process controlled via Rhubarb can be quite computationally intensive, and is the current bottleneck for performance.
+- Blink frequency is chosen using a random probability
```
Comment (vnajack) [inline README.md:111]: Is blinking based on probability a limitation or is it just something we can refine after peer evaluations?

File: README.md [hunk 4]
```diff
@@ -104,11 +100,13 @@ charge of mapping spoken language from an audio file into a series of phonemes,
...
+- Avatar poses are chosen at random.
```
Comment (vnajack) [inline README.md]:
```suggestion
- Avatar poses are chosen at random. A suggestion for future development would be to include sentiment analysis on the text and choose appropriate poses: however, this also depends on which poses are provided by the artist.
```

**Review by vnajack — APPROVED** [review]
I added a few suggestions, but it looks great! Thank you for updating the README. It was pretty out-of-date, so this is looking better!

File: README.md (hunk 3 above)
Comment (LoganParker) [reply README.md:111]: Yeah good point, something we can figure out for sure. I'd say it's not necessarily a limitation but definitely something we should review

**Review by mkuelker — APPROVED** [review]
Looks Good!

**Pull request merged**